commit/plugin.go

```diff
@@ -129,7 +133,7 @@ func NewPlugin(
 
 	return &Plugin{
 		donID:               donID,
-		nodeID:              nodeID,
+		nodeID:              oracleID,
```

Comment:
Would be nice to rename this field in the struct to `oracleID` to be consistent.

Reply:
will do in a different pr